scan

```diff
@@ -80,13 +80,15 @@ def scan_domains(scanners, domains):
             'writer': scanner_writer
         }
 
-    # For each domain, run through each scanner.
```

Comment:
The loop structure was flipped so I could keep the same instance of phantomjs open while scanning.